bundles/binding/org.openhab.binding.zwave/database/aeon/zw056.xml

```diff
@@ -1,119 +1,123 @@
+	            	<Default>2</Default>
+			<Size>1</Size>
+	            	<Minimum>0</Minimum>
+	            	<Maximum>100</Maximum>
+	            	<Label lang="en">Repetitions of ringtone</Label>
+			<Help lang="en">If the time of playing doorbell ringtone is more than 20s, the volume of ringtone will reduce to silence no matter how many repetitions of doorbell ringtone are set. Default=2</Help>
+		</Parameter>
+	        <Parameter>
+			<Index>5</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Size>1</Size>
+			<Minimum>1</Minimum>
+			<Maximum>100</Maximum>
+			<Label lang="en">Default ringtone</Label>
+			<Help lang="en">Set the default doorbell ringtone. Default=1</Help>
+		</Parameter>
+		<Parameter>
+			<Index>6</Index>
+			<Type>long</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Minimum>0</Minimum>
+			<Maximum>100</Maximum>
+			<Label lang="en">Ringtone to play</Label>
+			<Help lang="en">Set a ringtone to play. Default=0</Help>
+		</Parameter>
+		<Parameter>
+			<Index>7</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Ignore</Label>
+			</Item>
+	                <Item>
+				<Value>1</Value>
+				<Label lang="en">Play</Label>
+	                </Item>
+	                <Item>
+				<Value>2</Value>
+				<Label lang="en">Stop</Label>
+	                </Item>
+	                <Item>
+				<Value>3</Value>
+				<Label lang="en">Pause</Label>
+	                </Item>
+	                <Item>
+				<Value>4</Value>
+				<Label lang="en">Next</Label>
+	                </Item>
+	                <Item>
+				<Value>5</Value>
+				<Label lang="en">Previous</Label>
+	                </Item>
+	            	<Label lang="en">Send Basic Set on open/close event</Label>
+			<Help lang="en">Default=0</Help>
+	        </Parameter>
+	        <Parameter>
+			<Index>8</Index>
+	            	<Type>int</Type>
+	            	<Default>10</Default>
+	            	<Size>1</Size>
+	            	<Minimum>0</Minimum>
+	            	<Maximum>10</Maximum>
+	            	<Label lang="en">Ringtone volume</Label>
+			<Help lang="en">Default=10</Help>
+	        </Parameter>
+	        <Parameter>
+			<Index>80</Index>
+	            	<Type>list</Type>
+	            	<Default>0</Default>
+	            	<Size>1</Size>
+	                <Item>
+	                    	<Value>0</Value>
+	                    	<Label lang="en">Nothing</Label>
+	                </Item>
+	                <Item>
+	                    	<Value>1</Value>
+	                    	<Label lang="en">Hail CC</Label>
+	                </Item>
+	                <Item>
+	                    	<Value>2</Value>
+	                    	<Label lang="en">Basic CC Report</Label>
+	                </Item>
+			<Label lang="en">Enable Notifications</Label>
+			<Help lang="en">Enable to send notifications to associated devices (Group1) when the state of Siren changed (0=nothing,1=hailCC,2=basic CC report).</Help>
+		</Parameter>
```

Comment:
Yes, mostly the stuff for remapping the buttons on the front of the unit. Thought I would leave them alone as the user manual does have instructions for setting volume and ringtone based on the type of button press, switching them via configuration settings would make those portions of the manual incorrect, thought that could be confusing for someone that changed the parameters, then pulled out the manual a few months later and it didn't work right. Was trying to get the config items that can't be changed via the front buttons.